web/src/constants.ts

```diff
@@ -19,7 +19,7 @@ function podStatusIsCrash(status: string) {
   return status === podStatusError || status === podStatusCrashLoopBackOff
 }
 
-function podStatusIsError(status: string) {
+function podStatusErrorFunction(status: string) {
```

Comment:
I would rename this to `podStatusError` because you can tell by its type that it's a function

```suggestion
function podStatusError(status: string) {
```